sonaranalyzer-dotnet/its/expected/Ember-MM/Ember.Plugins-{9496C697-5AFD-4813-AEDC-AF33FACEADF0}-S2259.json

```diff
@@ -1,17 +0,0 @@
-{
-"issues":  [
-{
-"id":  "S2259",
-"message":  "'className' is null on at least one execution path.",
-"location":  {
-"uri":  "sources\Ember-MM\Ember.Plugins\PluginSectionHandler.cs",
```

Comment:
[Here](https://github.com/SonarSource/sonar-dotnet/blob/master/sonaranalyzer-dotnet/its/sources/Ember-MM/Ember.Plugins/PluginSectionHandler.cs#L91) we used to raise a null pointer exception. HOWEVER, it was an FP because

- `InvalidConfigException` which can be thrown when `className` is still null is caught by the previous `catch`.
- when `UnknownClassException` is thrown, we are sure that `className` is not null because of line 66